src/LoginContainer/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import './style.css';
+import { loginUser } from '../actions/user';
+
+
+    class LoginContainer extends React.Component{  
+      constructor(props) {
+          super(props);
+           this.state = {
+                email:"",
+                password:null,
+                data:""
+          };
+      }
+      myChangeHandler = (event) => {
+          var name = event.target.name;
+          var value = event.target.value
+          this.setState({[name]:value})      
+      } 
+      onSubmit = (e) => {
+          e.preventDefault();
+          const {email,password } = this.state;
+          this.props.buyCake(email,password)
+      }   
+     render(){
+        const loading = this.props.login.login_loading
+        if(loading){
+          alert(loading);
```

Comment:
Remove alert, they're not friendly to see. instead, you can a empty <div>loading...</div>